starter-code/src/clue.js

```diff
@@ -1,94 +1,214 @@
+}
+
+
+const missScarlet = {
+    first_name: "Kasandra",
+    last_name: "Scarlet",
+    color: "red",
+    description: "She is an A - list movie star with a dark past",
+    age: 31,
+    image: "https: //www.radiotimes.com/uploads/images/Original/111967.jpg",
+    occupation: "Actor"
+};
+
+const mrsPeacock = {
+    first_name: "Eleanor",
+    last_name: "Peacock",
+    color: "blue",
+    description: "She is from a wealthy family and uses her status and money to earn popularity",
+    age: 36,
+    image: "https: //metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation: "Socialité"
+};
+
+
+const mrMustard = {
+    first_name: "Jack",
+    last_name: "Mustard",
+    color: "yellow",
+    description: "He is a former football player who tries to get by on his former glory",
+    age: 62,
+    image: "https: //static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+    occupation: "Retired Football player"
+};
+
 
 // Weapons
 
-name: rope        ---   weight: 10
-name: knife       ---   weight: 8
-name: candlestick ---   weight: 2
-name: dumbbell    ---   weight: 30
-name: poison      ---   weight: 2
-name: axe         ---   weight: 15
-name: bat         ---   weight: 13
-name: trophy      ---   weight: 25
-name: pistol      ---   weight: 20
+const rope = {
+    name: "rope",
+    weight: 10
+};
+
+const knife = {
+    name: "knife",
+    weight: 8
+};
+
+const candlestick = {
+    name: "candlestick",
+    weight: 2
+};
+
+const dumbbell = {
+    name: "dumbbell",
+    weight: 30
+};
+
+const poison = {
+    name: "poison",
+    weight: 2
+};
+
+const axe = {
+    name: "axe",
+    weight: 15
+};
+
+const bat = {
+    name: "bat",
+    weight: 13
+};
+
+const trophy = {
+    name: "trophy",
+    weight: 25
+};
+
+const pistol = {
+    name: "pistol",
+    weight: 20
+};
+
+
 
 // Rooms
-name: Dining Room
-name: Conservatory
-name: Kitchen
-name: Study
-name: Library
-name: Billiard Room
-name: Lounge
-name: Ballroom
-name: Hall
-name: Spa
-name: Living Room
-name: Observatory
-name: Theater
-name: Guest House
-name: Patio
-*/
+const diningRoom = {
+    name: "Dining Room"
+};
+const conservatory = {
+    name: "Conservatory"
+};
+
+const kitchen = {
+    name: "Kitchen"
+};
+
+const study = {
+    name: "Study"
+};
+
+const library = {
+    name: "Library"
+};
+
+const billiardRoom = {
+    name: "Billiard Room"
+};
+
+const lounge = {
+    name: "Lounge"
+};
+
+const ballroom = {
+    name: "Ballroom"
+};
+
+const hall = {
+    name: "Hall"
+};
+
+const spa = {
+    name: "Spa"
+};
+
+const livingRoom = {
+    name: "Living Room"
+};
+
+const observatory = {
+    name: "Observatory"
+};
+
+
+const theater = {
+    name: "Theater"
+};
+
+const guestHouse = {
+    name: "Guest House"
+};
+
+const patio = {
+    name: "Patio"
+};
+
+
 
 // Characters Collection
 const charactersArray = [];
+charactersArray.push("mrGreen", "drOrchid", "profPlum", "missScarlet", "mrsPeacock", "mrMustard");
 
```

Comment:
Here you are pushing strings inside the array, not the actual variable that withholds the object as a value. The Jasmin Test is just passed , cause its not actually using those arrays. So you're passing the test. But normally this would not work;)..so push it without quotes.